### ISSUE TITLE
NameError: name 'log2' is not defined while calling psnrb()

### ISSUE THREAD

**Issue by GuidoBartoli** [andrewekhalel/sewar#17]
Hi,
Here are the steps to reproduce the error, it seems that you forgot `from math import log2` on top of `utils.py`:
```
import cv2 as cv
x = cv.imread('/home/user/Pictures/a.jpg', cv.IMREAD_GRAYSCALE)
y = cv.imread('/home/user/Pictures/b.jpg', cv.IMREAD_GRAYSCALE)
import sewar
sewar.psnrb(x, y)

Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/user/.virtualenvs/sq/lib/python3.6/site-packages/sewar/full_ref.py", line 387, in psnrb
    bef = _compute_bef(P)
  File "/home/user/.virtualenvs/sq/lib/python3.6/site-packages/sewar/utils.py", line 144, in _compute_bef
    t = log2(block_size)/log2(min(height, width))
NameError: name 'log2' is not defined
```

**Comment by kumarsanjeet1234** [comment]
Yes I do get the same issues

**Comment by aayushsin** [comment]
I guess, it needs to be changed to np.log2 from log2. Can anyone from the authors include this?

File: utils.py, line 144

**Comment by andrewekhalel** [comment]
Thank you guys for reporting this. Fixed on e3fde6f7deab47b98812aace115fe9e0324c6a00
